cms/utils/helpers.py

```diff
@@ -109,19 +109,3 @@ def is_editable_model(model_class):
     except KeyError:
         return False
     return isinstance(admin_class, FrontendEditableAdminMixin)
-
-
-def get_admin_model_object_by_id(model_class, obj_id):
```

Comment:
This was an early attempt at not just blindly patching the codebase. I prefer the new manager idea, much cleaner.